benchmark-bench-prerelease.sh

```
@@ -0,0 +1,49 @@
+# The job must be executed under a TRT-LLM clone root folder
+
+env && hostname && nvidia-smi
+start_time=$(date '+%Y-%m-%d-%H:%M:%S')
+output_folder=benchmark.run.${SLURM_JOB_ID}.$(date '+%Y-%m-%d-%H:%M:%S')
+mkdir -p ${output_folder} && chmod 777 ${output_folder} # the docker user is root, otherwise it can not write logs to this folder
+
+run_benchmark() {
+    IMAGE="urm.nvidia.com/sw-tensorrt-docker/tensorrt-llm-staging/release:main-x86_64"
+    docker run --rm --ipc=host --ulimit memlock=-1 --ulimit stack=67108864 \
+        -v /home/scratch.trt_llm_data:/home/scratch.trt_llm_data:ro \
+        -v `pwd`:`pwd` \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)